packages/gestalt/src/Dropdown.js

```diff
@@ -157,19 +151,20 @@ export default function Dropdown({
     let numItemsRendered = 0;
     const items = [];
 
-    const dropdownChildrenArray = Children.toArray(children);
+    dropdownChildrenArray.forEach((child) => {
+      const subSectionChildren = child.props.children;
+      const childrenDisplayName = child.type.displayName;
```

Comment:
tiniest of nits that I think "childDisplayName" is technically more accurate (cuz the childrens' display name is actually DropdownItem

Reply:
DONE